site/src/modules/provisioners/ProvisionerAlert.tsx

```diff
@@ -4,30 +4,38 @@ import { AlertDetail } from "components/Alert/Alert";
 import { Stack } from "components/Stack/Stack";
 import { ProvisionerTag } from "modules/provisioners/ProvisionerTag";
 import type { FC } from "react";
+
+export enum AlertVariant {
+	Standalone = "Standalone",
+	InLogs = "InLogs",
+}
+
 interface ProvisionerAlertProps {
 	title: string;
 	detail: string;
 	severity: AlertColor;
 	tags: Record<string, string>;
+	variant?: AlertVariant;
 }
 
 export const ProvisionerAlert: FC<ProvisionerAlertProps> = ({
 	title,
 	detail,
 	severity,
 	tags,
+	variant = AlertVariant.Standalone,
 }) => {
 	return (
 		<Alert
 			severity={severity}
-			css={(theme) => {
-				return {
+			{...(variant === AlertVariant.InLogs && {
```

Comment:
nit (readability): I think you can extract it to a separate function or move up to a const.

Reply:
done